_episodes/00-intro.md

```diff
@@ -5,6 +5,7 @@ questions:
 - "How is using a cluster different than your laptop?"
 keypoints: 
 - "A cluster or distributed system is useful when your computations require more than one computer." 
+- "Because a cluster is distributed, it is only useful for certain types of computational problems."
```

Comment:
I think there are two distinct points are being brought up by @pdoehle, and that we should address them explicitly:

- Not all problems can be solved faster using a cluster (i.e., the problem must expose some kind of parallelism)
- Programs or applications generally need to be written in a special way to take advantage of cluster hardware (i.e., the software must be able to take advantage of this parallelism)